package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mail",
+  "version": "0.0.1",
```

Comment:
I used "npm init" to create the skeleton. 0.1 showed a error. Don't know if it is important.